src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,43 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int bodyWidth;
```

Comment:
Consider adding an access modifier to the 'bodyWidth' field. It's good practice to encapsulate the fields of a class and expose them through getters and setters.